crates/bundler/src/bundler.rs

```diff
@@ -78,4 +101,142 @@ impl Bundler {
 
         Ok(tx_hash)
     }
+
+    // TODO: add more relay endpoints support
+    /// Send a bundle as Flashbots bundles
+    #[allow(clippy::needless_return)]
+    #[allow(clippy::clone_double_ref)]
+    pub async fn send_next_bundle_flashbots(&self, uos: &Vec<UserOperation>, test: Option<bool>) -> anyhow::Result<H256> {
```

Comment:
The `test` param could be removed, here. Move this kind of config in the entrypoint config.